### DISCUSSION TITLE
Restrictions incurred by not using 2FA

### DISCUSSION THREAD

**Discussion by phikal** [community/community#65333] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The recent email says

> # How will this affect my account?

> On October 12th, 2023 at 00:00 (UTC) your account will be required to have 2FA for authentication. If you have not yet enrolled by that date, **your ability to access GitHub.com will be limited until you finish the enrollment process. **

What does the emphasized text imply exactly?  My account will persist, but I won't be able to log in?  Will I be able to respond to Emails if someone pings me?

**Comment by 3urobeat** [comment]
Hey!  
You can read more about the change in this blog post: https://github.blog/2023-03-09-raising-the-bar-for-software-security-github-2fa-begins-march-13/  
To address your question directly, it says that after the deadline you won't be able to login anymore until you activate 2fa. I suspect you will still be able to reply to emails but no guarantee.  
I'd recommend setting up 2FA either way as it is done quickly and drastically strengthens the security of your account.

**Comment by phikal** [comment]
3urobeat ***@***.***> writes:

> Hey!  

Hello,

> You can read more about the change in this blog post: https://github.blog/2023-03-09-raising-the-bar-for-software-security-github-2fa-begins-march-13/  
> To address your question directly, it says that after the deadline you
> won't be able to login anymore until you activate 2fa. I suspect you
> will still be able to reply to emails but no guarantee.

I couldn't find any further details in that blog post.  Not being able
to log in would be acceptable to me, but if I cannot respond to Emails
my account would be pointless.

> I'd recommend setting up 2FA either way as it is done quickly and drastically strengthens the security of your account.

I don't really care about security, so this is just an inconvenience to
me that I am not interesting in taking up.  It will eventually just lock
me out of my account anyway, in my experience.

**Comment by corwinn** [comment]
> What does the emphasized text imply exactly?

To me it implies, that an eventual attacker would be able to take over my account by activating the mandatory clean fresh air, after just breaking my password - the same one I won't be able to change anymore as a security measure.

I haven't decided yet, whether it is preferable to just delete my account, than to let it be left at the mercy of wrong MFA actors.

In any case, the time will come when all these hardware (pass)keys, etc. shall start failing (even due to damaged USB port(s)), get lost, etc. A lot of popcorn will be needed :)

Also, they have no interest whatsoever in explaining, in details, what that text implies. The future will show.
